.gitignore

```diff
@@ -65,4 +65,3 @@ tools/test.file.1G
 
 # autotest logs/data
 /.autotest
-
```

Comment:
Seems an unnecessary change? 😄

Reply:
I think he activated the `pre-commit`s for the project. They are fixing the file endings for me as well. We can deactivate that, does not seem that important.
https://github.com/rucio/rucio/blob/a4db9e66ba5a175a721f75bb59b0761c92e8db5a/.pre-commit-config.yaml#L5